### ISSUE TITLE
cannot detect changes of .coffee files on server-side and restart meteor

### ISSUE THREAD

**Issue by beeplin** [thereactivestack/meteor-webpack#72]
in server/entry.js:

require('./a.coffee');
require('./b.js');

while changes of `b.js` can cause a meteor restart, changes of `a.coffee` cannot.

on client side it's all okay.

**Comment by eXon** [comment]
CoffeeScript is currently not supported with webpack. CoffeeScript and TypeScript support will be added very soon :)

**Comment by beeplin** [comment]
actually it already supports coffeescript very well by settings in `webpack.conf.js` like this:
```
	module: {
		loaders: [
			{
				test: /\.coffee$/,
				loader: 'babel!coffee',
			} 
		]
	}
```
of coz `coffee-loader` must be installed via npm.

up to now all my coffee codes both in client and in server are compiling very well through webpack 1.0.3. the only problem is the server-side hot-restart.

**Comment by eXon** [comment]
Yea. To do that, you need a Meteor compiler that use the extesion. I'm on it :)

**Comment by eXon** [comment]
Should be fixed on 1.0.4 :)
